apps/etterlatte-api/src/main/kotlin/no/nav/etterlatte/behandling/sak/BehandlingSakRoutes.kt

```diff
@@ -52,6 +52,18 @@ fun Route.behandlingSakRoutes(
             }
         }
 
+        route("oms/har_loepende_sak") {
```

Comment:
Ser vi har litt ulik fremgangsmåte for _case styles_ i koden. Men jeg tror kebab case er å foretrekke m.a.o endre til `oms/har-loepende-sak`